tests/test_examples.py

```diff
@@ -63,37 +63,45 @@ def test_HELIOTRON_results(HELIOTRON):
     np.testing.assert_allclose(theta_err.mean(), 0, atol=2e-2)
 
 
+# TODO: confirm this function does what we want; are different grids used as the comment implied?
+#   then I can add test with helical and radial ForceBalance(node_pattern=quad) for codecov coverage
 def test_force_balance_grids():
     """Compares radial & helical force balance on same vs different grids."""
 
```

Comment:
I feel like this function (just look at `master` branch) doesn't change any grids like it says it does

Reply:
On `master`, the `RadialForceBalance` defaults to a concentric grid with "cos" rotation, while the `HelicalForceBalance` default to "sin" rotation. The `ForceBalance` objective uses the same rotation for both grids to reduce the number of computations. 

On this branch we have removed the `rotation` argument from the `ConcentricGrid` class, so now this test is now obsolete. (We virtually always use the combined `ForceBalance` objective anyways.). But a user could still compute the radial & helical errors on separate grids if they assign those grids themselves.